### DISCUSSION TITLE
Fusion openDTU keine Verbindung mit FritzBox

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#2085] [Q&A]
Hallo,

idh betreibe einen HMS-1000 mit einer Fusion openDTU, aktuelle FW ist drauf.
Das funktioniert (eigentlich) prima, allerdings habe ich jetzt ein Problem entdeckt, das abhängig ist von der **Einschaltreihenfolge (!)** von DTU und WLAN.

(Zum Hintergrund:
Ich schalter nachts die DTU und das WLAN aus, daher erfolgt morgend dann das Einschalten von DTU und WLAN.)

**1. So funktioniert es immer:**
- Zunächst sind WLAN und DTU ausgeschaltet
- Dann **zuerst WLAN** einschalten, etwas warten bis WLAN komplett läuft
- Dann DTU einschalten
- DTU-App starten => nach einigen Sekunden erscheint die DTU-Oberfläche korrekt mit allen Daten auf dem Handy.

**2. Das funktioniert nie:**
- Zunächst sind WLAN und DTU ausgeschaltet
- Dann **zuerst** **DTU** einschalten (klar, die ist vom Handy noch nicht erreichbar, da ja noch kein WLAN an)
- Nach ca. 1 Minute WLAN einschalten und warten
   => es wird **keine** Verbindung aufgebaut, die DTU ist vom Handy über WLAN nicht erreichbar
- In der FritzBox 6591 cable auch nachgesehen unter "WLAN" => dort ist DTU mit WLAN als "nicht verbunden" gekennzeichnet
- Nun die DTU für ca. 30 sek. **aus- und wieder eingeschaltet** (WLAN bleibt ein)
- DTU-App starten => nach kurzer Zeit erscheint die DTU-Oberfläche korrekt mit allen Daten.

Ich habe keine Idee, wo dieses Problem herkommen könnte - kann jemand hier helfen?

Vielen Dank im Voraus und Grüße
Werner

**Comment by Fokker-1** [comment]
Hallo,

ich frage noch mal freundlich - gibt es eine Idee dazu?

Danke und Grüße
Werner

**Reply by ghost** [reply]
Hi,
nicht wirklich, keiner kennt deine Netzwerk- oder WLAN-Konfiguration.
Es gab kürzlich ähnliche Themen:
[DHCP funktioniert nicht.](#2076) Ob eine Lösung gefunden wurde, steht noch nicht da.
[OpenDTU mit Fritzbox verbunden aber nicht erreichbar.](#1074) Hier wurde das Problem an der Fritz!Box insbesondere dem Mesh vermutet, eine Rückmeldung über Erfolg steht noch aus.

**Comment by Fokker-1** [comment]
... erst mal danke, schade.

Ich kenne mich nicht so aus, aber kann ich irgenwie feststellen, ob die DTU überhaupt kontinuierlich Versuche unternimmt, sich mit dem WLAN der FB zu verbinden, nachdem das WLAN aktiviert wurde?

Ist das WLAN aktiviert bevor die DTU eingeschaltet wird, funktioniert ja alles.

Danke und Grüße
Werener

**Reply by ghost** [reply]
Ja, ist ein verzwickter Fehler.
Ich gehe davon aus, das die Versuche im der Debug-Konsole auftauchen.
Du kannst kannst ja die openDTU  aus den nicht verbundenen Geräten der FB werfen, bzw. die openDTU generell entfernen. Ich glaube, wenn sie dann mal einen Verbindungsversuch macht und dabei von der FB gesehen wird, taucht sie wieder in der Liste auf.

Mist: @tbnobody war schneller.

**Comment by tbnobody** [comment]
Die DTU versucht nicht ständig sich zu verbinden. Es wird alle 10min für 30 Sek. ein Verbindungsversuch probiert. Genaueres ist aber in der seriellen Konsole ersichtlich

**Reply by ghost** [reply]
In Bezug auf:
> "Es wird alle 10min für 30 Sek. ein Verbindungsversuch probiert."

Wie lange hast du gewartet?
Wen das WLAN erst 1 Minute oder später nach dem Einschalten der DTU da ist, macht das Nichtverbinden tatsächlich Sinn.
Der nächste Versuch kommt ja erst viel später, wie @tbnobody gerade eben ausgeführt hat.

**Comment by Fokker-1** [comment]
... ok., danke Dir für die Info.

Aber die Konsole sehe ich ja nicht, da ich nicht auf die DTU-Oberfläche komme.

**Reply by ghost** [reply]
AP-Modus
Suche ein WLAN in der Nähe mit Namen openDTUxxxxxxx und verbinde dich.
IP = Web Adresse: 192.168.4.1

**Comment by tbnobody** [comment]
https://www.opendtu.solar/firmware/howto/serial_console/

**Comment by Fokker-1** [comment]
... vielen Dank, ich versuche es mal weiter - mit meinen minimalistischen Kenntnissen  :-(

Es bleibt mir aktuell schleierhaft, wieso die **Reihenfolge (!)** des Einschaltens vom WLAN der FB und der DTU solche Folgen hat.

**Comment by Fokker-1** [comment]
... danke noch dafür:

https://www.opendtu.solar/firmware/howto/serial_console/

**Comment by Fokker-1** [comment]
Guten Morgen zusammen,

ich habe jetzt mal getestet hinsichtlich der Info "10 Minuten für 30 Sek."

Ja, das ist es! 

Da wäre ich nie drauf gekommen.
Allerdings frage ich mich nun, warum diese Verbindungsversuche nur alle 10 Minuten erfolgen, gibt es dafür einen speziellen Grund? 

ich kenne mich ja damit nicht aus, aber die DTU weiß ja, wenn sie keine WLAN.Verbindung zur FB hat, dann könnte die ja auch z. B. alle 1 Minute oder alle 30 Sek. einen neuen Versuch starten und nicht nur alle 10 Minuten?

Wäre prima, wenn mir hier jemand das erklären könnte  :-)

Noch mal DANKE an alle hier und Grüße
Werner

**Reply by tbnobody** [reply]
Das ist bereits in den FAQ beantwortet ![image](https://github.com/tbnobody/OpenDTU/assets/1753222/ab497fa9-7deb-4616-903e-a4569e65132d)

**Comment by Fokker-1** [comment]
Hallo,

vielen Dank für die ergänzenden Infos aus den  FAQ.
Das hat mir geholfen zum Verständnis.

Sorry für meine späte Rückinfo.

Danke und Grüße
Werner
